prow/git/v2/publisher.go

```diff
@@ -59,14 +68,49 @@ func (p *publisher) Commit(title, body string) error {
 	return nil
 }
 
-// ForcePush pushes the local state to the remote
+// ForcePush pushes the local state to the publish remote
 func (p *publisher) ForcePush(branch string) error {
-	p.logger.Infof("Pushing branch %q", branch)
-	remote, err := p.remote()
+	if err := p.PublishPush(branch, true); err != nil {
```

Comment:
this is a complicated way of doing `return p.PublishPush(branch, true)`

Reply:
I kept this for backward compability. I guess it should be fine to change this from the components that are using it already.

Reply:
No, my point is that

```
return p.PublishPush(branch, true)
```

is the exact same as
```
if err := p.PublishPush(branch, true); err != nil {
    return err
}
return nil
```

Except that the former is written in one line of code wheas what you did needs four